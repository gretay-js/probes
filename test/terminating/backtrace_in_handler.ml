let bt title =
  Printexc.get_callstack 9999
  |> Printexc.raw_backtrace_to_string
  |> Str.global_replace (Str.regexp_string " (inlined)") ""
  |> Printf.printf "%s:\n%s" title

let () =
  bt "Before";
  [%probe "backtrace" (bt "Inside")];
  bt "After"
