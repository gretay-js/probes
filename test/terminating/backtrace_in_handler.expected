Before is backtrace length > 2? true
Inside is backtrace length > 2? true
After is backtrace length > 2? true
